In [1]:
import math

In [3]:
math.log(1)

0.0

In [4]:
math.log10(1/10)

-1.0

In [5]:
math.log10(1000)

3.0

In [6]:
math.log10(10)

1.0

In [7]:
math.log10(pow(10,2))

2.0

When we have a log form such as: 

$$ log_{10} \frac{M}{N} $$ 

We can change it so it looks like: 

$$ log_{10} M - log_{10} N $$ 

When we have a log form such as: 

$$ log_{10} MN $$ 

We can change it so it looks like:

$$ log_{10}M + log_{10}N $$  

If we have 

$$ log_{10}M^k $$ 

We can change it so it looks like: 

$$ k log_{10} M $$ 

To express 472.3 in scientific notation, we simple do this:

$$ 4.723 x 10^2 $$ 

To express 0.041 in scientific notation, we can do:

$$ 4.1 \times 10^{-2} $$ 

We need to find the product of:

$$ 1.2 \times 10^{-2} * 3.65 \times 10^{8} $$ 

The first step is to multiply the decimals together: 

$$ 1.20 * 3.65 = 4.38 $$ 

The next step is to add the exponents of the 10.

$$ 10^{-2 + 8} = 10^{6} $$ 

Now we just put it together in scientific notation,

$$ 4.38 \times 10^6 $$ 

In [9]:
1.2 * 3.65 

4.38

In [16]:
def sciNoteProduct(b1,ten1, b2, ten2):
    newb = (b1 * b2) / 10
    newten = ten1 + ten2
    newNote = str(newb) + " " + "*" + " " + str(10) + "^" + str(newten+1)
    return newNote 

sciNoteProduct(5.2,-6,8,3)

'4.16 * 10^-2'

Find the quotient:

$$ 2300 \times (0.4 \times 10^{-8}) $$ 

Step 1: Change 2300 to scientific notation:

$$ 2300 = 2.3 \times 10^3 $$ 

Step 2: Divide the bases. 

$$ \frac{2.3}{0.4} = 5.75 $$ 

Step 4: Subtract the exponents of the two bases

$$ 10^{3 - (-8)} $$ 
$$ 10^{11} $$ 

Step 5: Put the base and 10 exponent together:

$$ 5.75 \times 10^{11} $$ 

In [15]:
2.3 / 0.4

5.749999999999999

In [18]:
def sciNoteQuotient(b1,ten1, b2, ten2):
    newb = (b1 / b2) * 10
    newten = ten1 - ten2
    newNote = str(newb) + " " + "*" + " " + str(10) + "^" + str(newten-1)
    return newNote 

sciNoteQuotient(5.2,6,8,-3)

'6.5 * 10^8'

1 minute has 60 seconds and 1 hour has 60 minutes. Using this information, we can do 60 x 60 to get the number of seconds in one hour. We can then transform this product into scientific notation.

$$ 60 * 60 = 3600 $$ 
$$ 3600 = 3.6 \times 10^3 seconds $$ 

There are 24 hours in one day so let's first transform 24 into scientific notation and then we can find the product of the two. 

$$ 24 = 2.4 \times 10^1 $$ 

$$ 3.6 * 2.4 = 8.64 $$ 

Then we need to add the exponents of the 10s.

$$ 10^{3 + 1} $$ 

Then we put the new base and new exponent together:

$$ 8.64 \times 10^4 $$ 

In [19]:
3.6 * 2.4

8.64

$$ \frac{3}{4}x = 179,000 $$ 

$$ \frac{3}{4} = 1.79 \times 10^5 $$

$$ \frac{3}{4} = 0.75 = 7.5 \times 10^{-1} $$

$$ (7.5 \times 10^{-1}) * x = 1.79 \times 10^5 $$

$$ x = \frac{1.79 \times 10^{5}}{7.5 \times 10^{-1}} $$ 

In [20]:
1.79 / 7.5

0.23866666666666667

Reshape the quotient in scientific notation.

$$ 0.24 = 2.4 \times 10^{-1} $$ 

Subtract the exponents. 

$$ 5 - (-1) $$ 

Put it altogether:

$$ 2.4 \times 10^{6-1} $$

$$ 2.4 \times 10^{5} $$ 

If the International Space Station is at an altitude of 385 km, that is our starting point, in other words it is a y-intercept because we would be 385 km at h = 0, where h is our input variable. 

The question is asking how many hours it will take for the International Space Station to reach a star that is 4.8 lightyears away. To do this, we need to first convert 4.8 lightyears to kilometers so we are in a similar unit as the ISS's starting altitude. We also need to know the rate of travel which is given to us as 27,707 km per hour. Using this information, we can create a linear equation as follows:

new Altitude = Rate of Travel per Hour * Number of Hours + Starting Altitude 

We'll represent new Altitude as y, rate of travel per hour as m, number of hours as h, and starting altitude as b.

$$ y = mx + b $$ 

m = 27,707 km per hour, b = 385 km, the new altitude we are trying to approach is 4.8 lightyears which we can say is our y 

The first step is to change the rate of travel into scientific notation:

$$ 27,707 = 2.7707 \times 10^4 $$ 

The next step is to change 385 km to scientific notation:

$$ 385 = 3.85 \times 10^2 $$ 

Our y value is given to us in light years, but we need to convert it into kilometers. 

To do this we need to know the conversion rate between the two which is given to us in the problem:

$$ 1 light year = 9.5 \times 10^{12} km $$ 

If we are trying to get to a star that is 4.8 light years away from our current position, we need to convert 4.8 light years into km. 

4.8 light years = 

$$ 4.8 * 9.5 \times 10^{12} $$ 

In [21]:
4.8 * 9.5 

45.6

We need to reshape the product into scientific notation by making it:

$$ 45.6 = 4.56 \times 10^{1} $$

We can then put this exponent of 1 to be added to the 12.

$$ 4.56 \times 10^{13} $$ 

Now that we have 4.8 light years in kilometers, we can subtract the starting altitude and then divide by the rate of travel to get the number of hours needed to reach the star.

